CHANGELOG.unreleased.md

```diff
@@ -18,6 +18,7 @@
 - [VM/Snapshot export] Fix `Error: no available place in queue` on canceling an export via browser then starting a new one when the concurrency threshold is reached [#5535](https://github.com/vatesfr/xen-orchestra/issues/5535) (PR [#5538](https://github.com/vatesfr/xen-orchestra/pull/5538))
 - [Servers] Hide pool's objects if its master is unreachable [#5475](https://github.com/vatesfr/xen-orchestra/issues/5475) (PR [#5526](https://github.com/vatesfr/xen-orchestra/pull/5526))
 - [Host] Restart toolstack: fix `ECONNREFUSED` error (PR [#5553](https://github.com/vatesfr/xen-orchestra/pull/5553))
+- [VM migration] Show an error when a SR is not selected (PR [#5568](https://github.com/vatesfr/xen-orchestra/pull/5568))
```

Comment:
I'm not sure the changelog entry is necessary. It's not precise enough because you *can* migrate without selecting a SR, and explaining the exact issue might be too technical for the changelog.